docs/GettingStarted.md

```diff
@@ -29,10 +29,14 @@ task components:build-all
 
 *off-chain*
 
-_STATUS: TODO_
+Test a specific contract off-chain
+```bash
+task test:contract-off-chain CONTRACT=<contract-name>
```

Comment:
`task test:contract-off-chain CONTRACT=tg-contract-payments` gives an error running it.

Could you test this and give me and example how to do it? (There is only one contract here I think)